src/stcal/resample/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .resample import (
+    OutputTooLargeError,
+    Resample,
+    compute_wcs_pixel_area,
+    UnsupportedWCSError,
+)
+
+__all__ = [
+    "OutputTooLargeError",
```

Comment:
This custom error and all associated code is being removed from JWST in build 11.2, so it should be removed here as well; see https://github.com/spacetelescope/jwst/pull/8975